test/e2e/specs/editor/various/toolbar-roving-tabindex.spec.js

```diff
@@ -96,6 +96,7 @@ test.describe( 'Toolbar roving tabindex', () => {
 		pageUtils,
 	} ) => {
 		await editor.insertBlock( { name: 'core/table' } );
+		await page.keyboard.press( 'ArrowLeft' );
```

Comment:
Immediately after the table is inserted, the focus is on the input number field (Column Count).
Pressing the up or down key in this state will respect the upper limit of the number (which is a native event), so press the left key in advance to give focus to the block wrapper.